x-pack/test/common/services/security_solution/roles_users_utils.ts

```diff
@@ -8,8 +8,8 @@
 import {
   KNOWN_ESS_ROLE_DEFINITIONS,
   KNOWN_SERVERLESS_ROLE_DEFINITIONS,
-  SecurityRoleName,
 } from '@kbn/security-solution-plugin/common/test';
+import type { SecurityRoleName } from '@kbn/security-solution-plugin/common/test';
```

Comment:
Thanks for the change. There are couple more places that needs updating import as type for `SecurityRoleName`